components/hybrid_engine/src/observer.rs

```diff
@@ -44,14 +44,15 @@ impl Observer {
         coprocessor_host
             .registry
             .register_admin_observer(priority, BoxAdminObserver::new(self.clone()));
+        // Evict cache when a peer applies snapshot.
```

Comment:
BTW, please add some comments about why we have to evict region before applying snapshot.

Reply:
Done